pypi/data-processing/requirements.txt

```diff
@@ -22,3 +22,7 @@ opencc-python-reimplemented==0.1.7
 selectolax==0.3.17
 openai==1.3.7
 python-docx==1.1.0
+
+bs4==0.0.1
+playwright=1.40.0
```

Comment:
@GHOSTHSP  You have introduced new dep `playwright `. What is this for?

Reply:
To handle websites built with React 、Vue or ... , simulate a browser to scrape web page data